jack.py

```diff
@@ -1895,6 +2023,8 @@ def _hasflag(self, flag):
         return bool(self._code & flag)
 
 
```

Comment:
I just update my rep with all modifications.

I finish a first version of my application. I'm able to play several 
loop (6 in tests) with jack configured with a ~40 ms latency. No xrun 
appears. With latency at 11ms there is some xruns around 10 per minute.

I have not notice any performance issue.

Julien

Le 29/04/2015 16:32, Matthias Geier a écrit :
>
> In jack.py 
> <https://github.com/spatialaudio/jackclient-python/pull/10#discussion_r29341095>:
>
> > @@ -1895,6 +2023,8 @@ def _hasflag(self, flag):
> >          return bool(self._code & flag)
> >
> >
>
> There should be only 2 blank lines.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/spatialaudio/jackclient-python/pull/10/files#r29341095>.
>